tools/topology/sof/pipe-eq-fir-volume-capture.m4

```diff
@@ -0,0 +1,129 @@
+# Passthrough with volume Pipeline and PCM
```

Comment:
Even if this would not be used for demo topologies please keep this included in your PR. It will be useful later.